portfolio/src/main/java/com/google/sps/servlets/golfJoke.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns HTML that shows answer to joke */
+@WebServlet("/Golf-Joke")
+
+public class golfJoke extends HttpServlet {
```

Comment:
In Java, you use pascal case when naming classes, so `golfJoke` should be `GolfJoke`